build/squire-raw.js

```diff
@@ -1626,16 +1629,12 @@ var firstOrLastLine = function(self){
     var rect = range.getBoundingClientRect()
     var nodeOffset = rect.top
     var parentOffset = parentBlock.getBoundingClientRect().top
-    if(nodeOffset === parentOffset){
-      return {firstLine: true, lastLine: true}
-    }
     nodeOffset = nodeOffset - parentOffset
```

Comment:
the problem I think comes from my assumption that the parentBlock would be the editor itself